content/blog/2020/11/2020-11-08-help-test-jenkins.adoc

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Help Us Test Jenkins"
+tags:
+- jenkins
+- testing
+- ui
+author: markewaite
+description: "Latest Jenkins weekly releases contain large changes.  Help us test the most recent releases"
+opengraph:
+  image: /images/post-images/2020-11-08-help-test-jenkins/help-test-jenkins.png
```

Comment:
Two-level list does not look well IMHO, I would remove UI/Core separation.

Reply:
Revised in 701c32ef454ad17e3e637001c69db7483c8ff625

Reply:
Thanks